src/ngx_http_auth_jwt_module.c

```diff
@@ -670,6 +672,55 @@ static char *get_jwt(ngx_http_request_t *r, ngx_str_t jwt_location)
       jwtPtr = ngx_str_t_to_char_ptr(r->pool, jwtCookieVal);
     }
   }
+  else if (jwt_location.len > sizeof(QUERY_STRING_PREFIX) && ngx_strncmp(jwt_location.data, QUERY_STRING_PREFIX, sizeof(QUERY_STRING_PREFIX) - 1) == 0) {
+    jwt_location.data += sizeof(QUERY_STRING_PREFIX) - 1;
+    jwt_location.len -= sizeof(QUERY_STRING_PREFIX) - 1;
 
+    size_t token_key_start = 0, token_value_start = 0, token_end = 0;
+
+    bool found_token = search_token_from_args(
+      &jwt_location,
+      &r->args,
+      &token_key_start,
+      &token_value_start,
+      &token_end
+    );
+
+    if (!found_token) return NULL;
```

Comment:
Please wrap in brackets and add an `else` containing the rest of the function. This aids in readability and follows our existing style in the repo.

Reply:
Thanks, fixed in commits: https://github.com/TeslaGov/ngx-http-auth-jwt-module/pull/107/commits/7dd8b005b8e932175e7cef4f34c026eb1b9c6fdf and https://github.com/TeslaGov/ngx-http-auth-jwt-module/pull/107/commits/17fbdc0c5bc5f587cefe710c5ce164d2c810d096.

Without the early return replacing the else-block, I changed to:
```c
if (token_found)
{
  do_stuff();
}
```